test/common/index.js

```diff
@@ -1059,6 +1059,10 @@ const common = {
     return hasOpenSSL(3, 2);
   },
 
+  get hasOpenSSL34() {
+    return hasOpenSSL(3, 4);
+  },
```

Comment:
When I added `hasOpenssl()` in https://github.com/nodejs/node/pull/53456, the idea was to stop adding new functions for every OpenSSL version to `common`, and instead directly call, e.g.`hasOpenSSL(3, 4)` in the test.